include/log.h

```diff
@@ -84,9 +88,10 @@ inline int Log(const char *msg, ...)
 		va_list ap;
 		gettimeofday(&tv,NULL);
 #if defined(__linux__)
+		pid_t tid = gettid();
 		char name[16];
 		pthread_getname_np(pthread_self(), name,sizeof(name));
-		printf("[%-16s][%.10ld.%.3ld][LOG]", name, (long)tv.tv_sec, (long)tv.tv_usec / 1000);
+		printf("[%-16s][0x%-4x][%.10ld.%.3ld][LOG]", name, tid, (long)tv.tv_sec, (long)tv.tv_usec / 1000);
 #else
 		printf("[0x%lx][%.10ld.%.3ld][LOG]", (long)pthread_self(), (long)tv.tv_sec, (long)tv.tv_usec / 1000);
```

Comment:
even if it will provide no extra info, please add it also here so we have logs vertically alligned

Reply:
Do you mean change this line?
```suggestion
		printf("[0x%lx][0x%-4x][%.10ld.%.3ld][LOG]", (long)pthread_self(), tid, (long)tv.tv_sec, (long)tv.tv_usec / 1000);
```
This wouldn't be used when __linux__ is defined, would it? Plus, the gettid() is only applicable to Linux.

> Function: pid_t gettid (void)(https://sourceware.org/glibc/manual/html_mono/libc.html#index-gettid)
> Preliminary: | MT-Safe | AS-Safe | AC-Safe | See [POSIX Safety Concepts](https://sourceware.org/glibc/manual/html_mono/libc.html#POSIX-Safety-Concepts).
> 
> The gettid function returns the thread ID of the current thread. The returned value is obtained from the Linux kernel and is not subject to caching. See the discussion of thread IDs above, especially regarding reuse of the IDs of threads which have exited.
> 
> This function is specific to Linux.

Reply:
ups, my bad, somehow I thought it was a normal if..